website/blog/2020-06-30-data-lake-analytics-blog.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Data Lake Analytics: Alibaba's Federated Cloud Strategy
+author: George Wang
+authorURL: https://www.linkedin.com/in/george-wang-9a5a46190/
+---
+
+Presto is known to be a high-performance, distributed SQL query engine for Big Data. It offers large-scale data analytics with multiple connectors for accessing various data sources. This capability enables the Presto users to further extend some features to build a large-scale data federation service on cloud. <br>
+<br>
+Alibaba Data Lake Analytics embraces Presto’s federated query engine capability and has accumulated a number of successful business use cases that signify the power of Presto's analytics capability.
+<br><br>
+
+
+<!-- truncate -->
+
+## Data Federation Service
+
+Today’s cloud database empowers cloud users to easily and directly interact with other cloud products and services, available to be built for their own cloud data lake analytic scenarios and solutions. The architecture to pile up a cloud-based data lake analytic scenario is divided into three layers:
+
+<br><br>
+![](/img/blog/2020-06-30-data-lake-analytics-blog/threeLayerCloud_v1.png)
+<br><br>
+**First Layer**: A variety of selected cloud native data storage and database services. 
+<br><br>
+Users store and place their data on a series of cloud-native data storage services and cloud databases systems. Cloud data sources vary depending on different business use cases and business scenarios, but speaking in general, there are two cloud data storage: NoSQL and relational databases. Various choices of storage service may include: cost-effective object cloud storage which may store files, structured, semi-structured, unstructured raw data, multimedia files; NoSQL, RDS, and other cloud database services;
+<br><br>
+
+**Second Layer**: Cloud native data lake analysis service layer.
+<br><br>
+This layer emphasizes a very important feature in Cloud Native computing, that is, Serverless is server-free and is the basic service for building SaaS on the cloud, except for Data Lake analysis scenarios, there is also a more ubiquitous serverless PaaS and Function-computing as-a-Service (FaaS) such as Alibaba Cloud Function Calculation. This article focuses on the serverless analysis capability of Data Lake analytic scenarios.<br>
+
+1. Elasticity - This allows an on-demand flexibility capability, which means being flexible in time, predictable and intelligent mixed load handling ability<br>
+
+2. Resiliency - User’s unperceived High Availability capability includes transparent rolling upgrade, fast handover, disaster tolerance across AZ (available area), etc<br>
```

Comment:
"unperceived High Availability capability" => not clear what this means. Can you please update/revise?

"available area" => availability zone

Reply:
fixed